### PR TITLE
refactor(organization): good naming practice

### PR DESCRIPTION
**Pull request by ChanyutJean**

### CODE REVIEW

File: proto/hts/organizer/service.proto [hunk 1]
```diff
@@ -96,50 +91,50 @@ message HasEventRequest {
...
 
 service OrganizerService {
   rpc CreateOrganization(CreateOrganizationRequest) returns (hts.common.Result);
-  rpc ReadOrganization(UserRequest) returns (ReadOrganizationResult);
-  rpc ReadOrganizationById(ReadByIdRequest) returns (ReadOrganizationByIdResult);
+  rpc GetOrganization(UserRequest) returns (ReadOrganizationResponse);
+  rpc GetOrganizationById(ReadByIdRequest) returns (ReadOrganizationByIdResponse);
   rpc UpdateOrganization(UpdateOrganizationRequest) returns (hts.common.Result);
-  rpc DeleteOrganization(DeleteOrganizationRequest) returns (hts.common.Result);
+  rpc RemoveOrganization(DeleteOrganizationRequest) returns (hts.common.Result);
```
Comment (trif0lium) [inline proto/hts/organizer/service.proto]: `DeleteOrganizationRequest` -> `RemoveOrganizationRequest`

File: proto/hts/organizer/service.proto [hunk 2]
```diff
@@ -96,50 +91,50 @@ message HasEventRequest {
...
   rpc UpdateEventDuration(UpdateEventDurationRequest) returns (hts.common.Result);
-  rpc DeleteEvent(DeleteEventRequest) returns (hts.common.Result);
+  rpc RemoveEvent(DeleteEventRequest) returns (hts.common.Result);
   rpc UpdateRegistrationRequest(UpdateRegistrationRequestRequest) returns (hts.common.Result);
   rpc AddTag(UpdateTagRequest) returns (hts.common.Result);
   rpc RemoveTag(UpdateTagRequest) returns (hts.common.Result);
-  rpc ReadTag(UserRequest) returns (ReadTagResult);
-  rpc ReadTagById(ReadByIdRequest) returns (ReadTagByIdResult);
+  rpc ReadTag(UserRequest) returns (ReadTagResponse);
+  rpc ReadTagById(ReadByIdRequest) returns (ReadTagByIdResponse);
```
Comment (trif0lium) [inline proto/hts/organizer/service.proto]: `ReadTagById` -> `GetTagById`

File: proto/hts/organizer/service.proto [hunk 3]
```diff
@@ -96,50 +96,51 @@ message HasEventRequest {
...
 
 service OrganizerService {
   rpc CreateOrganization(CreateOrganizationRequest) returns (hts.common.Result);
-  rpc ReadOrganization(UserRequest) returns (ReadOrganizationResult);
-  rpc ReadOrganizationById(ReadByIdRequest) returns (ReadOrganizationByIdResult);
+  rpc GetOrganization(UserRequest) returns (GetOrganizationResponse);
+  rpc GetOrganizationById(GetByIdRequest) returns (GetOrganizationByIdResponse);
   rpc UpdateOrganization(UpdateOrganizationRequest) returns (hts.common.Result);
-  rpc DeleteOrganization(DeleteOrganizationRequest) returns (hts.common.Result);
+  rpc RemoveOrganization(DeleteOrganizationRequest) returns (hts.common.Result);
```
Comment (ChanyutJean) [inline proto/hts/organizer/service.proto]:
```suggestion
  rpc RemoveOrganization(RemoveOrganizationRequest) returns (hts.common.Result);
```

**Review by boompttn — APPROVED** [review]
LGTM

**Pull request merged**